app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command.split()) != 3:
+        return
+
+    command_line, start, end = command.split()
```

Comment:
don't split the command twice
create a separate variable for it and call it on demand